xml/System.Diagnostics/ActivityLink.xml

```diff
@@ -132,7 +132,15 @@ Activity links can be used to represent batched operations where an activity was
         <summary>Indicates whether the current activity link is equal to another activity link.</summary>
         <returns>
           <see langword="true" /> if the current activity link is equal to <paramref name="other" />; otherwise, <see langword="false" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+    ## Remarks
+
+    <xref:System.Diagnostics.ActivityLink> equality is currently undefined. The current behavior is two ActivityLinks with the same ActivityContext and no tags will return true for Equals. Prior to .NET 7 ActivityLinks sharing an ActivityTagsCollection instance and the same ActivityContext would return true for Equals. After .NET 7 Equals will always return false when tags are used because the behavior is undefined. This may change in the future. Exercise caution in calling this method.
```

Comment:
```suggestion
    <xref:System.Diagnostics.ActivityLink> equality is currently undefined. The current behavior is two ActivityLinks with the same ActivityContext and no tags will return true for Equals. Prior to .NET 7 ActivityLinks sharing an ActivityTagsCollection instance and the same ActivityContext would return true for Equals. Starting from .NET 7, Equals will always return false when tags are used. This may change in the future. Exercise caution in calling this method.
```